src/Card/Card.scss

```diff
@@ -422,6 +422,8 @@ a .pgn__card {
   border-radius: 0 0 .375rem .375rem;
```

Comment:
[curious] Should we be using the `$card-border-radius` SCSS variable here to be consistent with border radius throughout the `Card` (sub-)components?